src/VSTS-Bot.Api/Dialogs/RootDialog.cs

```diff
@@ -154,10 +154,14 @@ public RootDialog(Uri eulaUri, TelemetryClient telemetryClient)
             {
                 await result;
             }
-            catch (Exception e)
+            catch (UnknownCommandException)
```

Comment:
That way unknown command would not be reported back to user.

Reply:
If there is an unknown command from a dialog, most likely the user wants to exit the flow. So we route them back to the starting point.
If then still it is unknown, the help list is shown.